enterprise/server/util/oci/oci.go

```diff
@@ -36,6 +38,11 @@ import (
 	bspb "google.golang.org/genproto/googleapis/bytestream"
 )
 
+const (
+	tagLRUMaxEntries = 1000
+	tagLRUExpiration = time.Minute * 15
```

Comment:
nit: the names could be a little clearer; also probably worth adding comments since they are tunable parameters.

```suggestion
	resolveImageDigestLRUMaxEntries = 1000
	resolveImageDigestLRUExpiration = time.Minute * 15
```